cf-check/backup.c

```diff
@@ -106,6 +106,11 @@ int backup_files(Seq *filenames)
 int backup_main(int argc, const char *const *const argv)
 {
     Seq *files = argv_to_lmdb_files(argc, argv);
+    if (files == NULL || SeqLength(files) == 0)
+    {
+        Log(LOG_LEVEL_ERR, "No database files to back up");
+        return -1;
```

Comment:
As mentioned in Slack, I realized this `-1` gets propagated as an exit code of the process and that's not good, exit codes should be non-negative.

Reply:
Thanks! I usually cherry-pick whole PRs so they have the matching title. I added the cherry-pick label to the follow-up PR, so I wont forget. https://github.com/cfengine/core/pull/3769